config.yml.erb

```diff
@@ -523,7 +523,7 @@ db_proxy_admin:
 db_writer: !Secret
 reporting_db_writer: <%=db_writer%>
 
-dashboard_db_name: <%= "dashboard#{_env}#{_test_env_num}" %>
+dashboard_db_name: <%= "dashboard#{env}#{_test_env_num}" %>
```

Comment:
Should it not be `<%= "dashboard_#{env}#{_test_env_num}" %>`

Reply:
done!